manifests/p/PolyMC/PolyMC/1.3.0/PolyMC.PolyMC.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: PolyMC.PolyMC
+PackageVersion: 1.3.0
+PackageLocale: en-US
+Publisher: PolyMC

```

```suggestion
Publisher: PolyMC Contributors
```
This must match Control Panel.